### PR TITLE
SED-2821 my account layout changes

### PR DESCRIPTION
**Pull request by lukasz-lepa**
https://exense.atlassian.net/browse/SED-2821

### CODE REVIEW

File: projects/step-frontend/src/lib/modules/admin/components/my-account/my-account.component.html [hunk 1]
```diff
@@ -1,77 +1,78 @@
...
-    </step-table>
   </mat-card-content>
 </mat-card>
 <mat-card>
-  <mat-card-header>
-    <mat-card-title>Role</mat-card-title>
+  <mat-card-header class="header-with-button">
+    <mat-card-title class="title-with-hint">
+      <div>Api key</div>
+      <step-description-hint [description]="apiKeyDescription"></step-description-hint>
```
Comment (neogucky) [inline projects/step-frontend/src/lib/modules/admin/components/my-account/my-account.component.html:27]: I know that this is not consequently followed in the App (I will put it up for discussion on Monday), but I prefer to have texts defined in the template file.

File: projects/step-frontend/src/lib/modules/admin/components/my-account/my-account.component.ts [hunk 2]
```diff
@@ -52,10 +58,15 @@ export class MyAccountComponent implements OnInit, OnChanges, OnDestroy {
...
   private _dialogs = inject(DialogsService);
+  private credentialsUrl =
+    'https://step.exense.ch/knowledgebase/userdocs/userpreferences/#user-credentials-and-api-keys';
+  private preferencesUrl = 'https://step.exense.ch/knowledgebase/userdocs/userpreferences/#preferences';
 
   readonly canChangePassword = !!this._authService.getConf()?.passwordManagement;
   readonly canGenerateApiKey = !!this._authService.getConf()?.authentication;
   readonly preferences$ = new BehaviorSubject<KeyValue<string, string>[]>([]);
+  readonly apiKeyDescription = `API keys can be used for authentication with the step API (<a href="${this.credentialsUrl}">knowledgebase</a>)`;
+  readonly preferencesDescription = `Add key/value pairs of custom preferences (<a href="${this.preferencesUrl}">knowledgebase</a>)`;
```
Comment (neogucky) [inline projects/step-frontend/src/lib/modules/admin/components/my-account/my-account.component.ts:69]: As mentioned prior please move both url and text into the template and add a target="_blank" to the a tags.

**Review by neogucky — APPROVED** [review]

File: projects/step-frontend/src/lib/modules/admin/components/my-account/my-account.component.scss [hunk 3]
```diff
@@ -35,4 +49,16 @@ step-my-account {
...
+    display: flex;
+    align-items: center;
+    padding-left: 0.5rem;
+    font-size: 1.4rem;
+  }
+
+  .title-with-hint {
+    display: flex;
+    flex-direction: row;
+  }
```
Comment (dvladir) [inline projects/step-frontend/src/lib/modules/admin/components/my-account/my-account.component.scss:63]: The same could be achieved by adding some styles in title container only
```suggestion
  .title-with-hint {
    display: flex;
    line-height: 2rem
    gap: .5rem;
  }
```

**Review by dvladir — CHANGES_REQUESTED** [review]

File: projects/step-frontend/src/lib/modules/admin/components/my-account/my-account.component.scss (hunk 3 above)
Comment (lukasz-lepa) [reply projects/step-frontend/src/lib/modules/admin/components/my-account/my-account.component.scss:63]: "Preferences" and '?' icon are not aligned anymore with the styles above

image

**Review by neogucky — APPROVED** [review]

**Review by dvladir — APPROVED** [review]

**Pull request merged**